SoftLayer/CLI/firewall/detail.py

```diff
@@ -19,12 +19,24 @@ def cli(env, identifier):
     mgr = SoftLayer.FirewallManager(env.client)
 
     firewall_type, firewall_id = firewall.parse_id(identifier)
+    result = mgr.get_instance(firewall_id)
+
+    table = formatting.KeyValueTable(['name', 'value'])
+    table.align['name'] = 'r'
+    table.align['value'] = 'l'
+
+    table.add_row(['id', result['id']])
```

Comment:
use either `result.get()` or `utils.lookup()` (for the nested dictionary items) for all of these add_row entries. If, for some strange reason, these fields are not returned from the API they cause hard crashes of the slcli which we want to avoid.